### PR TITLE
Gcp arm

### PR DESCRIPTION
**Pull request by paigerube14**
Adding workflow for gcp-ovn-arm configuration

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-arm-3nodes

File: ci-operator/step-registry/openshift-qe/installer/gcp/ipi/ovn/arm/openshift-qe-installer-gcp-ipi-ovn-arm-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-qe-installer-gcp-ipi-ovn-arm
+  steps:
+    env: 
+      OCP_ARCH: arm64
+    pre:
+    - ref: gcp-provision-vpc
```
Comment (qiliRedHat) [inline ci-operator/step-registry/openshift-qe/installer/gcp/ipi/ovn/arm/openshift-qe-installer-gcp-ipi-ovn-arm-workflow.yaml:7]: @paigerube14 I think for a gcp ipi install without vpc, some steps in pre are not necessary. Like in cucushift-installer-rehearse-gcp-ipi-ovn-provision, there are only the following steps
  - chain: ipi-conf-gcp
  - ref: ovn-conf 
  - chain: ipi-install
  - ref: enable-qe-catalogsource
  - chain: cucushift-installer-check

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-arm-3nodes

File: ci-operator/step-registry/openshift-qe/installer/gcp/ipi/ovn/arm/openshift-qe-installer-gcp-ipi-ovn-arm-workflow.yaml (hunk 1 above)
Comment (paigerube14) [reply ci-operator/step-registry/openshift-qe/installer/gcp/ipi/ovn/arm/openshift-qe-installer-gcp-ipi-ovn-arm-workflow.yaml]: I copied the wrong steps, updated now thanks!

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-arm-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-arm-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-arm-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-arm-control-plane-arm-3nodes

**Comment by paigerube14** [comment]
@svetsa-rh @qiliRedHat PTAL passing job here: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/50110/rehearse-50110-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-arm-control-plane-arm-3nodes/1772974713123901440

**Pull request merged**